packages/shared/components/popups/LedgerTransactionPopup.svelte

```diff
@@ -0,0 +1,69 @@
+<script lang="typescript">
+    import { onDestroy } from 'svelte'
+    import { Animation, KeyValueBox, Text } from 'shared/components'
+    import { localize } from '@core/i18n'
+    import {
+        resetLedgerSendConfirmationProps,
+        resetLedgerMintNativeTokenConfirmationProps,
+        ledgerSendConfirmationProps,
+        ledgerMintNativeTokenConfirmationProps,
+    } from '@core/ledger'
+    import { formatHexString } from '@core/utils'
+    import { openPopup } from '@lib/popup'
+
+    export let toAddress: string
+    export let toAmount: string
+    export let hash: string
+
+    const hasSendConfirmationProps = (toAddress && toAmount) || hash
```

Comment:
We'll have to adjust this logic for native token minting, but it should be done in the corresponding PR.